dask/dask.yaml

```diff
@@ -11,7 +11,7 @@ dataframe:
   shuffle-compression: null  # compression for on disk-shuffling. Partd supports ZLib, BZ2, SNAPPY
   parquet:
     metadata-task-size-local: 512  # Number of files per local metadata-processing task
-    metadata-task-size-remote: 16  # Number of files per remote metadata-processing task
+    metadata-task-size-remote: 1  # Number of files per remote metadata-processing task
```

Comment:
Setting this greater than 1 is only an optimization when you are dealing with many small files. However, it is more likely that we will need to parse metadata when files are large, and we need to split them by row-group. For large remote files, `metadata_task_size=1` is often a better choice.